src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -635,18 +636,23 @@ class DialogmoteService(
         personIdentNumber: PersonIdentNumber,
         token: String,
     ) {
-        val tilfelleStart = oppfolgingstilfelleClient.oppfolgingstilfelle(
+        val tilfelle = oppfolgingstilfelleClient.oppfolgingstilfelle(
             callId = callId,
             personIdentNumber = personIdentNumber,
             token = token,
-        )?.fom ?: throw RuntimeException("Cannot create MoteStatusEndring: No TilfelleStart was found")
+        ) ?: throw RuntimeException("Cannot create MoteStatusEndring: No Oppfolgingstilfelle was found")
+
+        if (dialogmoteStatus == DialogmoteStatus.INNKALT && tilfelle.isInactive()) {
+            throw RuntimeException("Cannot create MoteStatusEndring with status INNKALT: Dialogmoteinnkalling for person with inactive Oppfolgingstilfelle is not allowed")
```

Comment:
Ender dette opp som en forståelig respons til veileder?

Reply:
Nei, det er ikke kodet noe i Syfomodiaperson. Denne PR'en er kun for å hindre at et en uønsket dm-innkalling blir gjennomført. Det er ikke gjort noe for å hindre/informere veileder om at de ikke skal gjøre dette.